nil/cmd/nil_block_generator/internal/commands/client_test.go

```diff
@@ -0,0 +1,124 @@
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/NilFoundation/nil/nil/services/nilservice"
+	"github.com/NilFoundation/nil/nil/internal/collate"
+	"github.com/NilFoundation/nil/nil/common/logging"
+	"github.com/NilFoundation/nil/nil/tests"
+	"github.com/stretchr/testify/suite"
+	"github.com/NilFoundation/nil/nil/tools/solc"
+	"github.com/rs/zerolog"
+	"os"
+)
+
+const contract_str =
+"pragma solidity ^0.8.0;\n" +
```

Comment:
You can use multiline strings or something from our test contracts (https://github.com/NilFoundation/nil/tree/main/nil/contracts/solidity/tests)

Reply:
i prefer to keep unit tests independent

Reply:
Your test is not actually "unit" because you run nil service. It's pure integration test.